dev-util/ragel/ragel-7.0.0.10.recipe

```diff
@@ -0,0 +1,88 @@
+SUMMARY="Compiles executable finite state machines from regular languages"
+DESCRIPTION="Ragel compiles executable finite state machines from regular \
+languages. Ragel targets C, C++, Objective-C, C#, D, Java, Ruby, OCaml and Go.
+Ragel state machines can not only recognize byte sequences as regular \
+expression machines do, but can also execute code at arbitrary points in the \
+recognition of a regular language.
+Code embedding is done using inline operators that do not disrupt the regular \
+language syntax."
+HOMEPAGE="https://www.colm.net/open-source/ragel/"
+COPYRIGHT="2001-2016 Adrian Thurston
+	2007 Victor Hugo Borja
+	2011 Josef Goettgens"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://www.colm.net/files/ragel/ragel-$portVersion.tar.gz"
+CHECKSUM_SHA256="40562bcac66a22dbea8357a35745bbcb1ab596c262d8691145ee11aafa6f8dec"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandSuffix=$secondaryArchSuffix
+	commandBinDir=$binDir
+else
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	ragel$secondaryArchSuffix = $portVersion
+	cmd:ragel$commandSuffix = $portVersion compat >= 7.0
+	lib:libfsm$secondaryArchSuffix = $portVersion compat >= 0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	colm$secondaryArchSuffix
```

Comment:
You should be more specific. Does this port depend on `colm` as a whole or only `lib:libcolm`?

Reply:
Just lib:libcolm